docs/se/webclient/introduction.adoc

```diff
@@ -233,4 +245,77 @@ requestBuilder.readerContext().registerReader(JsonSupport.reader()); // <2>
 requestBuilder.request(JsonObject.class)
 ----
 <1> Adds JSON-P writer only to this request.
-<2> Adds JSON-P reader only to this request.
\ No newline at end of file
+<2> Adds JSON-P reader only to this request.
+
+=== WebClient TLS setup
+
+Configure TLS either programmatically or by the Helidon configuration framework.
+
+==== Configuring TLS in your code
+
+The one way to configure TLS in WebClient is in your application code.
+
+[source,java]
+----
+KeyConfig keyConfig = KeyConfig.keystoreBuilder()
+                //Whether this keystore is also trust store
+                .trustStore()
+                //Keystore location/name
+                .keystore(Resource.create("client.p12"))
+                //Password to the keystore
+                .keystorePassphrase("password")
+                .build();
+
+WebClient.builder()
+         .tls(WebClientTls.builder()
+               .certificateTrustStore(keyConfig)
+               .clientKeyStore(keyConfig)
+               .build())
+         .build();
+----
+
+==== Configuring TLS in the config file
+
+It is also possible to configure TLS via the config file.
```

Comment:
Another way to configure TLS in WebClient is through the `application.yaml` configuration file.